core/app/services/uffizzi_core/deployment_service.rb

```diff
@@ -134,6 +134,15 @@ def build_preview_url(deployment)
       "#{deployment.subdomain}.#{Settings.app.managed_dns_zone}"
     end
 
+    def build_predictable_url(deployment)
```

Comment:
This is a wrong name for the method. For example it's going to be a regular URL if this condition is met:
```
return build_preview_url(deployment) if repo_name.nil? || pull_request_number.nil?
```